sms_wsj/database/utils.py

```diff
@@ -150,10 +153,17 @@ def scenario_map_fn(
             padded signal.
             pad and/or cut the source signal to match the length of the
             observations. Considers the offset.
+        add_speech_image:
+            The speech_image is always computed, but when it is not needed,
+            this option can reduce the memory consumption.
         add_speech_reverberation_early:
-            Calculate the add_speech_reverberation_early signal.
+            Calculate the speech_reverberation_early signal.
```

Comment:
Maybe we should add the definitions for `early` and `tail` signal, as well (e.g. early: first 50ms of reverberation,...)

Reply:
Ok, I added more text and described the early and late part